packages/@webex/plugin-meetings/src/meeting/request.ts

```diff
@@ -180,6 +183,10 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       },
     };
 
+    if (alias) {
+      body.alias = alias;
+    }
```

Comment:
Can't we simply set it in the body itself? If it's not passed in, it'll be undefined anyway.

Reply:
```alias``` should not be sent to backend if not passed in. 

Same as ```locale``` and ```deviceCapabilities```.